### PR TITLE
update ga_squad example

### PR DESCRIPTION
**Pull request by PurityFan**
1. fix a bug on config_pai.yml and modified the default value of memory
2. Supplement the README.md doc of running ga_squad example on pai

### CODE REVIEW

File: examples/trials/ga_squad/README.md [hunk 1]
```diff
@@ -30,7 +32,7 @@ chmod +x ./download.sh
 ./download.sh
 ```
 
-## Download manually
+### Download manually
```
Comment (xuehui1991) [inline examples/trials/ga_squad/README.md]: after we add the script into command, do we still need to download manually?

File: examples/trials/ga_squad/README.md [hunk 2]
```diff
@@ -75,12 +77,70 @@ In the "trial" part, if you want to use GPU to perform the architecture search,
...
 `trialConcurrency` is the number of trials running concurrently, which is the number of GPUs you want to use, if you are setting `gpuNum` to 1.
 
-## submit this job
+### submit this job
 
 ```
 nnictl create --config ~/nni/examples/trials/ga_squad/config.yml
 ```
 
+## Run this example on pai
```
Comment (xuehui1991) [inline examples/trials/ga_squad/README.md]: OpenPAI?

File: examples/trials/ga_squad/README.md (hunk 1 above)
Comment (PurityFan) [reply examples/trials/ga_squad/README.md]: I didn't add the script to config.yml
I divided this doc into two cases: running in local/remote and running on pai. This part belong's to run the example in local, I think it's better to download the data first and run trial.py for several times.

File: examples/trials/ga_squad/README.md (hunk 2 above)
Comment (PurityFan) [reply examples/trials/ga_squad/README.md]: Good suggestion!

Comment (PurityFan) [reply examples/trials/ga_squad/README.md]: Done~

File: examples/trials/ga_squad/README.md [hunk 3]
```diff
@@ -75,12 +77,70 @@ In the "trial" part, if you want to use GPU to perform the architecture search,
...
+Please change the default value to your personal account and machine information. Including `nniManagerIp`, `dataDir`, `outputDir`, `userName`, `passWord` and `host`.
+
+In the "trial" part, if you want to use GPU to perform the architecture search, change `gpuNum` from `0` to `1`. You need to increase the `maxTrialNum` and `maxExecDuration`, according to how long you want to wait for the search result.
+
+`trialConcurrency` is the number of trials running concurrently, which is the number of GPUs you want to use, if you are setting `gpuNum` to 1.
+
+### submit this job
+
+```
+nnictl create --config ~/nni/examples/trials/ga_squad/config_pai.yml
```
Comment (leckie-chn) [inline examples/trials/ga_squad/README.md]: `~/nni/examples/trials/ga_squad/config_pai.yml` assume that user clone nni under $HOME directory

Comment (PurityFan) [reply examples/trials/ga_squad/README.md]: Sorry, not clear about the question? It's just an example of the path for config_pai.yml, also consistent with other examples.

**Pull request merged**